consensus/src/main/kotlin/maru/consensus/qbft/adaptors/QbftBlockCreatorAdaptor.kt

```diff
@@ -0,0 +1,59 @@
+/*
+   Copyright 2025 Consensys Software Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package maru.consensus.qbft.adaptors
+
+import maru.consensus.BlockCreator
+import maru.core.Seal
+import org.hyperledger.besu.consensus.qbft.core.types.QbftBlock
+import org.hyperledger.besu.consensus.qbft.core.types.QbftBlockCreator
+import org.hyperledger.besu.consensus.qbft.core.types.QbftBlockHeader
+import org.hyperledger.besu.consensus.qbft.core.types.QbftExtraDataProvider
+import org.hyperledger.besu.crypto.SECPSignature
+
+/**
+ * Adaptor to convert a [BlockCreator] to a [QbftBlockCreator].
+ */
+class QbftBlockCreatorAdaptor(
```

Comment:
> In the United States, “adapter” is preferred whereas in other English-speaking countries like the UK and Australia, “adaptor” is more common

I respect, that, but should we stick to "Adapter" to align with GoF's terminology 🙂?

Reply:
Yes, should use the US spelling